apps/bbsync/save.py

```diff
@@ -0,0 +1,59 @@
+from collectors.bzimport.collectors import BugzillaConnector
+from osidb.models import Flaw
+
+from .query import BugzillaQueryBuilder
+
+
+class BugzillaSaver(BugzillaConnector):
+    """
+    Bugzilla flaw bug save handler
+    flaw validity is assumed and not checked
+    """
+
+    class UnsaveableFlaw(Exception):
+        """
+        error caused by attempt to save a flaw which cannot be saved
+        either by its nature or due to the current saver capabilities
+        """
+
+        pass
+
+    def __init__(self, flaw):
+        """
+        init stuff
+        """
+        self.flaw = flaw
+
+    def save(self):
+        """
+        generic save serving as class entry point
+        which calls create or update handler to continue
+        returns an updated flaw instance (without saving)
+        """
+        return self.create() if self.flaw.bz_id is None else self.update()
+
+    def create(self):
+        """
+        create flaw in Bugilla
+        """
+        bqb = BugzillaQueryBuilder(self.flaw)
```

Comment:
I would be for some more descriptive name, `bz_query_builder` maybe?

Reply:
I can provide a longer name. As it is just used here and on the next line I did not consider it necessary when the meaning can be read from the class name but no problem to put it there again :smile_cat: